source/query/limit.txt

```diff
@@ -0,0 +1,67 @@
+.. _query-bar-limit:
+
+======================================
+Limit the Number of Returned Documents
+======================================
+
+.. default-domain:: mongodb
+
+If the query bar has the :guilabel:`Limit` option, you can specify the
+maximum number of documents to return.
+
+Set Documents to Return
+-----------------------
+
+To specify the limit:
+
+#. Click :guilabel:`Options`.
+
+#. Enter an integer representing the number of documents to return into
+   the :guilabel:`Limit` field.
+
+#.
+   .. include:: /includes/run-query.rst
```

Comment:
I think we should move the Note to be under the screenshot. It feels a bit awkward to have the step ("3"), then a semi-related note, then the screenshot associated with the step.